.github/workflows/unified-labeler.yml

```
@@ -0,0 +1,172 @@
+name: Pull Request Labeler
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, closed]
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+
+jobs:
+  labeler:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Create size labels if missing
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const labels = [
+              { name: 'size/XS', color: '0e8a16', description: 'Very small PR (≤10 lines)' },
+              { name: 'size/S', color: '28a745', description: 'Small PR (11–50 lines)' },
+              { name: 'size/M', color: 'fbca04', description: 'Medium PR (51–100 lines)' },
+              { name: 'size/L', color: 'd93f0b', description: 'Large PR (101–200 lines)' },
+              { name: 'size/XL', color: 'b60205', description: 'Very large PR (>200 lines)' }
```

Now that you’re unifying this workflow, I’d like to slightly adjust these values. Would it be possible to leave them like this:

```suggestion
              { name: 'size/M', color: 'fbca04', description: 'Medium PR (51–150 lines)' },
              { name: 'size/L', color: 'd93f0b', description: 'Large PR (151–250 lines)' },
              { name: 'size/XL', color: 'b60205', description: 'Very large PR (>250 lines)' }
```

This gives a bit more margin for PRs with extensive functionality.

We’ll also need to update this everywhere it’s used (including the PR template).